pkg/manifests/manifests.go

```diff
@@ -3599,3 +3643,65 @@ func hashStringMap(m map[string]string) string {
 	}
 	return hashByteMap(byteMap)
 }
+
+func validateLabelsAllowListFormat(value string) error {
```

Comment:
Taken from: https://github.com/kubernetes/kube-state-metrics/blob/5b7a11eed724511405a0148026ee96ac426f3787/pkg/options/types.go#L237, to ensure the additional list can be validated.